.github/contributing.rst

```diff
@@ -89,15 +89,70 @@ Cleaning up existing tests
 
 Original collection of tests in the test suite contained a lot of files
 that did not conform to the requirements listed above, and should
-eventually be fixed. See `Clean up and reorder tests #41
-<https://github.com/nim-works/nimskull/issues/41>`_ issue on github for
-more context and relevant discussion.
+eventually be fixed. List of known issues that should be fixed includes,
+but not limited to:
+
+- Check if test name makes sense - t123123_b.nim does not make sense,
+  change it to something matching what is being tested. File names usually
+  refer to the
+
+- Reduce number of echo-based error testing. If you see direct echo in test
+  consider changing it to the `doAssert` check instead.
+
+  Added assertions should replace original sequence of checks with
+  `doAssert a == <expected>` expression. If original check printed multiple
+  values in sequence (for example in a for loop) you can collect them into
+  a `seq[string]` variable and compare using `==` later.
+
+- Linking relevant issues in the test description (description: field) or
+  in comments.
+
+  Huge number of original tests "referred" to issue numbers using file
```

Comment:
This bit isn't very actionable unlike the rest, I would drop, rephrase, or indicate it as informational.